test/vanilla/complexTypesTests.ts

```diff
@@ -379,22 +394,22 @@ describe('typescript', function () {
           {
             'fishtype': 'sawshark',
             'species': 'dangerous',
-            'picture': new Buffer([255, 255, 255, 255, 254]),
+            'picture': new Uint8Array([255, 255, 255, 255, 254]),
             'length': 10,
             'age': 105
           }
         ]
-      };
+      });
       it('should throw when required fields are omitted from polymorphic types', function (done) {
-        testClient.polymorphism.putValidMissingRequired(badfish, function (error, result) {
+        testClient.polymorphism.putValidMissingRequired(getBadfish(), function (error, result) {
           should.exist(error);
           error.message.should.containEql('birthday');
           error.message.should.containEql('cannot be null or undefined');
           done();
         });
       });
 
-      var rawSalmon: AutoRestComplexTestServiceModels.SmartSalmon = {
+      var getRawSalmon: () => AutoRestComplexTestServiceModels.SmartSalmon = () => ({
```

Comment:
Why did this change?

Reply:
I wanted to be able to delete properties from this without worrying about the state of it in subsequent usages (e.g. doing a put after the checking the result of the get)